lib/server.js

```diff
@@ -160,6 +160,19 @@ module.exports = async config => {
         // Write prettier errors
         server.ext('onPreResponse', prettyPrintErrors);
 
+        // Log user
+        server.ext('onCredentials', (request, h) => {
+            const { username, scmContext, pipelineId } = request.auth.credentials;
+
+            if (pipelineId) {
+                logger.info(`[Login] Pipeline ${pipelineId}:${scmContext} ${request.method} ${request.path}`);
```

Comment:
I would omit the scmContext. how does the log appear for build token login ?

Reply:
username would be buildId for build token login
```
{"level":"info","message":"[Login] User 150 get /v4/pipelines/7/secrets","timestamp":"2022-11-03T22:05:13.294Z"}
```